### ISSUE TITLE
Function to move questions between pages

### ISSUE THREAD

**Issue by mgralikowski** [surveyjs/survey-creator#679]
### Are you requesting a feature, reporting a bug or ask a question?

Feature.

### What is the current behaviour?

No option to drag questions to another page or change parent page manually. For now, the only option is copy question to toolbox, remove from origin page, switch to target page, and use question, but it take some times and make some mess in toolbox.

### What is the expected behaviour?

D&d question to tab would be nice. Another option is manually change by select in properties.
We use pages as categories and moving questions between pages is missing function.

**Comment by andrewtelnov** [comment]
@mgralikowski There is a Page property in the Layout category.

Thank you,
Andrew

**Comment by mgralikowski** [comment]
Ok, so i am blind. So, I leave my proposal about d&d only - similar to panels.

**Comment by andrewtelnov** [comment]
Yes, Drag&Drop makes sense. We can change the current page during drag&drop,

Thank you,
Andrew
